src/components/SubNav/SubNav.stories.tsx

```diff
@@ -0,0 +1,493 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import Icon from "../Icons/Icon";
+
+import SubNav, {
+  SubNavButton,
+  SubNavLink,
+  actionBackgroundColorsArray,
+  highlightColorsArray,
+} from "./SubNav";
+
+const meta: Meta<typeof SubNav> = {
+  title: "Components/Page Layout/SubNav",
+  component: SubNav,
+  argTypes: {
+    className: { control: false },
+    actionBackgroundColor: {
+      control: "select",
+      options: actionBackgroundColorsArray,
+      defaultValue: { summary: "blogs" },
+    },
+    highlightColor: {
+      control: "select",
+      options: highlightColorsArray,
+      defaultValue: { summary: "blogs" },
+    },
+    id: { control: false },
+    selectedItem: { description: "Set item id to be selected." },
+    secondaryActions: {},
+    primaryActions: {},
+  },
+};
+
+export const WithControls: Story = {
+  args: {
+    className: undefined,
+    actionBackgroundColor: undefined,
+    highlightColor: undefined,
+    id: "sub-nav-id",
+    selectedItem: undefined,
+  },
+
+  render: (args: any) => (
+    <SubNav
+      {...args}
+      primaryActions={({ highlightColor, actionBackgroundColor }) => (
```

Comment:
I need to know more about why `highlightColor` and `actionBackgroundColor` are being passed here. I would like to simplify this so the prop value can be written like this:
```
primaryActions={
  <>
    ...
  </>
}
```

Reply:
@EdwinGuzman This is where I had concerns. Can the `primaryActions` and `secondaryActions` props be simplified for end-users?

Reply:
Because both props are render prop functions, a function has to be passed with the prop values that are passed from `SubNav`. It can't be simpler than this unfortunately.